packages/expo-video/ios/VideoManager.swift

```diff
@@ -50,17 +50,36 @@ class VideoManager {
 
   // MARK: - Audio Session Management
 
-  private func switchToActiveAudioSession() throws {
+  internal func setAppropriateAudioSessionOrWarn() {
     let audioSession = AVAudioSession.sharedInstance()
-    try audioSession.setCategory(.playback, mode: .moviePlayback)
-    try audioSession.setActive(true)
-  }
+    var audioSessionCategoryOptions: AVAudioSession.CategoryOptions = []
 
-  internal func switchToActiveAudioSessionOrWarn(warning: String) {
+    let isAnyPlayerPlaying = videoPlayers.allObjects.reduce(false) { result, object in
+      result || object.isPlaying
+    }
+    let areAllPlayersMuted = videoPlayers.allObjects.reduce(true) { result, object in
+      result && object.isMuted
+    }
+
+    // When all players are muted, or none are playing, we don't want to interrupt other audio sources on the device
+    if !isAnyPlayerPlaying || areAllPlayersMuted {
+      audioSessionCategoryOptions.insert(.mixWithOthers)
```

Comment:
so we have an audio session even though we're not pumping out any audio? do we need one then?

Reply:
I think the audio session always exists, we can only configure whether it's active and the category. I set the category every time, because a lot of the functionality depends on it (PiP, background playback) for example.

I guess this could be an issue if someone is playing audio with expo-audio or another package and then a muted expo-video changes the app's audio session category. I will update the code so that the category is only set when necessary.